docs/ru/development/developer-instruction.md

```diff
@@ -41,9 +41,15 @@ ClickHouse не работает и не собирается на 32-битны
 
 Выполните в терминале:
 
-    git clone git@github.com:your_github_username/ClickHouse.git --recursive
+    git clone --recursive --shallow-submodules git@github.com:your_github_username/ClickHouse.git
```

Comment:
We don't need to checkout submodules recursively for a while, so it's a good opportunity to remove it from the docs, since we are already here.

Reply:
```suggestion
    git clone --shallow-submodules git@github.com:your_github_username/ClickHouse.git
```